cli/command/network/connect.go

```diff
@@ -23,6 +23,7 @@ type connectOptions struct {
 	aliases      []string
 	linklocalips []string
 	driverOpts   []string
+	gwPriority   int
```

Comment:
I'm probably a bit late realising, but should this have been a uint, or do we expect negative values to be ok? 🙈

Reply:
It's an int on purpose as it allows to set the priority to a negative value when some existing endpoints have the default priority `0` set. @robmry wrote about that here: https://github.com/moby/moby/pull/48936#discussion_r1858809270

Reply:
But wasn't higher higher priority, so if I want a new network to become the default, I need to give it a higher priority?

Reply:
Absolutely. But if one of the current network give you a default gateway, and you want to stick with it, you can connect new networks with a negative priority.

Reply:
(mostly considering if it's zero, then I did not consider it a priority (and alphabetic order is used), and only if I would explicitly want a network to take priority, I'd probably set a non-zero value; any value lower (including zero) would not take priority in that  case)

Reply:
You might consider priority of existing endpoints only when you connect a new network. In that case, having the ability to set a negative priority gives you a way to not touch the existing endpoint(s), but still get the behavior you want.